setup.cfg

```diff
@@ -74,10 +74,8 @@ verbose = 2
 # https://pep8.readthedocs.io/en/latest/intro.html#error-codes
 format = pylint
 ignore =
-    E731  # do not assign a lambda expression, use a def
-    W503  # line break before binary operator
-    # because of YAPF - till https://github.com/google/yapf/issues/897 is resolved
```

Comment:
Does this comment not apply anymore?

Reply:
well we do not have Yapf check anymore :(

Reply:
We don't automatically but some people might still be running it manually

Reply:
I would hope for it

Reply:
@carmocca @Borda How should we do then?
1. keep ignoring E231 to avoid incompatibility between E231 and yapf (so close this PR), or
2. comply with E231 which is part of PEP8 (and merge this PR)

Reply:
yapf development looks dead, so doesn't look like https://github.com/google/yapf/issues/897 will be resolved anytime soon

if we ever apply formatting again, i'd reevaluate yapf considering the weird formatting it has shown to produce in many cases as well as it's lack of support. for this reason i'd go with (2)

But Jirka can have the final decision